src/browser_action/metric.js

```diff
@@ -152,17 +152,18 @@ export class Metric {
 
 export class LCP extends Metric {
 
-  constructor({local, finalized, background}) {
+  constructor({local, background}) {
     const thresholds = {
       good: 2500,
       poor: 4000
     };
 
+    local = local || 0;
```

Comment:
I'm not sure how this works with the extension, but technically LCP doesn't initialize at 0, since we don't report LCP if the page was loaded in the background — which is also true for all metrics except INP, where we're intentionally changing that behavior to make the metric less load focused.

Reply:
Added TODOs to consider better defaults. This is not actually new behavior as `vitals.js` was changed from defaulting all values to null from 0 so we can rip out the deprecated `finalized` logic.